upup/pkg/fi/cloudup/tests/bootstrapchannelbuilder/amazonvpc/manifest.yaml

```diff
@@ -147,7 +147,11 @@ spec:
   - id: k8s-1.16
     kubernetesVersion: '>=1.16.0'
     manifest: networking.amazon-vpc-routed-eni/k8s-1.16.yaml
+<<<<<<< HEAD
```

Comment:
Hmm, @MoShitrit it looks like something may have gone wrong with your CP.  Make sure to not have these merge diffs.

Reply:
Yep!
Just noticed that, working on fixing. 
That's the first time I used that `cherry_pick_pull.sh` script, must've missed something in the process.
Thanks for pointing that out!